lib/game.js

```diff
@@ -34,10 +36,14 @@ function drawOcean() {
   context.drawImage(images.ocean, 0, 135);
 };
 
+function drawObjects() {
+  canvasPainter.drawFish(fishies);
+}
+
 function generateFish() {
+  drawObjects();
```

Comment:
@stevepentler This got it working, but drawObjects shouldn't live in generateFish full time. Pull out to the game loop once the canvas painter can draw everything.